File: prob_1072.py
n = int(input())
dentro = 0
fora = 0

for numbers in range(n):
    x = int(input())
    if 10 <= x <= 20:
        dentro += 1
    else:
        fora += 1
print(f"{dentro} in\n{fora} out")

File: java/prob_1072.java
import java.util.Scanner;

public class prob_1072 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int quantidade, valor, dentro, fora;
        quantidade = sc.nextInt();
        dentro = 0;
        fora = 0;

        for (int i = 0; i < quantidade; i++){
            valor = sc.nextInt();
        
            if (valor >= 10 && valor <= 20)
                dentro++;
            else
                fora++;
        }

        System.out.println(dentro + " in");
        System.out.println(fora + " out");

        sc.close();
    }
}
